packages/odyssey-react/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -24,7 +24,7 @@ export default {
   },
 };
 
-const Template: Story<Props> = () => (
+const Template: Story<ComponentProps<typeof Tooltip>> = () => (
```

Comment:
Ok, I understand why, but this will be unfamiliar and cause friction for consumers. This is not idiomatic in other component libraries.

https://github.com/ant-design/ant-design/blob/master/components/button/button.tsx#L94
https://github.com/Shopify/polaris-react/blob/main/src/components/Button/Button.tsx#L23

Reply:
Yes, that sounds good.